hydrolib/tools/ext_old_to_new/converters.py

```diff
@@ -131,29 +190,77 @@ def convert(self, forcing: ExtOldForcing) -> Boundary:
 
         Args:
             forcing (ExtOldForcing): The contents of a single forcing block
-            in an old external forcings file. This object contains all the
-            necessary information, such as quantity, values, and timestamps,
-            required for the conversion process.
+                in an old external forcings file. This object contains all the
+                necessary information, such as quantity, values, and timestamps,
+                required for the conversion process.
+            start_time:
```

Comment:
I'd rename `start_time` to `time_unit`
`Formatted string containing the units of time, including absolute datetime reference information (according to UDunits). Example: "minutes since 1992-10-8 15:15:42.5 -6:00"
`

Reply:
renamed